src/Sparrow/Json/JsonOperationContext.cs

```diff
@@ -526,6 +526,13 @@ private unsafe LazyStringValue GetLazyString(StringSegment field, bool longLived
             }
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public unsafe LazyStringValue GetLazyStringValue(byte* ptr)
+        {
+            var size = BlittableJsonReaderBase.ReadVariableSizeInt(ptr, 0, out var offset);
```

Comment:
Bring the comment on the structure here.

Reply:
done